docs/upgrades/v0.14-to-v1-upgrade/examples/src/test/kotlin/examples/FunctionChanges.kt

```diff
@@ -0,0 +1,192 @@
+package examples
+
+import org.partiql.spi.function.PartiQLFunction
+import org.partiql.spi.function.PartiQLFunctionExperimental
+import org.partiql.types.function.FunctionParameter
+import org.partiql.types.function.FunctionSignature
+import org.partiql.value.Int32Value
+import org.partiql.value.PartiQLValue
+import org.partiql.value.PartiQLValueExperimental
+import org.partiql.value.PartiQLValueType
+import org.partiql.value.StringValue
+import org.partiql.value.float64Value
+import org.partiql.value.int32Value
+import org.partiql.value.stringValue
+import kotlin.math.abs
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class FunctionChanges {
+    @OptIn(PartiQLValueExperimental::class)
+    @Test
+    fun `defining function parameters`() {
+        // In PLK 0.14.9, function parameters use FunctionParameter with PartiQLValueType
+        // Creating parameters for function definition
+        val stringParam = FunctionParameter("text", PartiQLValueType.STRING)
+        val intParam = FunctionParameter("count", PartiQLValueType.INT)
+        assertEquals("text", stringParam.name) // Direct property access
+        assertEquals(PartiQLValueType.INT, intParam.type) // Direct property access
+    }
+
+    @OptIn(PartiQLValueExperimental::class)
+    @Test
+    fun `creating function signatures`() {
```

Comment:
nit: probably can separate this into separate examples. one for scalar one for aggregation